presto-product-tests/src/main/resources/sql-tests/testcases/window_functions/rowsRangeDifferentPartition.sql

```diff
@@ -0,0 +1,9 @@
+-- database: presto; groups: window;
+select orderkey, suppkey,
+discount,
+round(avg(discount) over (partition by suppkey order by orderkey rows between current row and unbounded following), 5)  avg_discount,
+extendedprice,
+round(avg(extendedprice) over (partition by discount range current row), 5) avg_extendedprice
```

Comment:
It's not very specific: even different range (with the same partition and order) is a different window.
Maybe: `sameFunctionOnDifferentPartitionOrderFrame`?